scripts/README.md

```diff
@@ -35,8 +35,8 @@ The following steps must be proposed as a pull request.
 - [ ] Checkout the last version of the working branch
 - [ ] Make sure you have the latest dependencies installed: `yarn`.
 - [ ] Build the packages: `yarn release:build`.
-- [ ] Release the versions on NPM: `yarn release:publish` (you need your 2FA device).
-- [ ] Create a new tag named with the release you just did `git tag v4.0.0-alpha.30 && git push upstream --tag`
+- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
+- [ ] Create a new tag named with the release you just did `git tag -a v4.0.0-alpha.30 -m "Version 4.0.0-alpha.30" && git push upstream --tag`
```

Comment:
Couldn't Core's releaseTag script be used instead?

Reply:
We could uniformize, but I won't have the time, I was already going above and beyond with this PR 😁. From my perspective, I don't really get what `releaseTag` adds to the table (feels overkill). 

As a side note, we could backport #3920

Reply:
> I don't really get what releaseTag adds to the table

Prevents typos, at the very least :)

> we could backport https://github.com/mui/mui-x/pull/3920 to the core.

In this case, I don't see what problem does having a checklist in the PR description solve.

Reply:
> Prevents typos, at the very least :)

Yeah ok 😁

> In this case, I don't see what problem does having a checklist in the PR description solve.

Personally, the main value for me was that it better explains what each step is about, I didn't notice the checkbox.